routing_common/car_model_coefs.hpp

```diff
@@ -20,13 +15,14 @@ HighwayBasedFactors const kHighwayBasedFactors = {
     {HighwayType::HighwayPrimary, InOutCityFactor(0.86, 0.89)},
     {HighwayType::HighwayPrimaryLink, InOutCityFactor(0.86, 0.89)},
     {HighwayType::HighwayResidential, InOutCityFactor(0.75)},
-    {HighwayType::HighwayRoad, InOutCityFactor(0.95)},
+    {HighwayType::HighwayRoad, InOutCityFactor(0.30)},
```

Comment:
These changes are accidental and should not be here.

Reply:
The changes were created by congfigure.sh, I'll revert them.